sklearn/naive_bayes.py

```diff
@@ -38,6 +38,25 @@
 class BaseNB(six.with_metaclass(ABCMeta, BaseEstimator, ClassifierMixin)):
     """Abstract base class for naive Bayes estimators"""
 
+    def _update_class_prior(self, class_prior=None):
```

Comment:
I would just move this into the GaussianNB class since it is not used anywhere else. (i.e if you want to keep it as a method)